schema/course.model.schema

```diff
@@ -338,6 +338,22 @@
                   "inputType": "Text",
                   "required": true,
                   "translatable": true
+                },
+                "mandatory": {
```

Comment:
mandatory vs required?

Reply:
@oliverfoster I think mandatory is the most commonly used. I guess it's a stronger word? I'll go with consensus though

Reply:
Stronger in what sense? More characters and syllables? `required` has a higher scrabble score at 18 vs `mandatory`'s 15.

Reply:
If words had muscles, Mandatory would have bigger ones

Reply:
If we go for `"compulsory": "Compulsory"` we could compel people to do the learning rather than just mandate it?

Reply:
I would have thought "required" to be the common use case but I have seen both use cases. The `default` can easily be amended per course when needed.